features/fixtures/news_and_communications_signup_content_item.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Can we not use the finder content item to get all this information?

Reply:
Yeah that might be a good optimisation. Especially as we request the content item for the finder in the email controller anyway. This would be an additional change to how it already works (and would require republishing a bunch of finders and unpublishing email signup content items) so I'm a bit reluctant to do it in this PR.